### PR TITLE
Trafodion 1894 & 1898

### PR DESCRIPTION
**Pull request by gtapper**
Fixes ready for delivery.

### CODE REVIEW

File: docs/client_install/src/asciidoc/_chapters/dbviz.adoc [hunk 1]
```diff
@@ -69,14 +66,16 @@ image:{images}/DbVisualizer_Driver_Manager.jpg[image]
 * Use the JDBC URL format:
 +
 ```
-jdbc:t4jdbc://<host-name or ip-address>:37800/:
+jdbc:t4jdbc://<host-name or ip-address>:23400/:
 ```
++
+*NOTE*: This example uses a modified port number (*37800*) rather than the default port number (*23400*).
```
Comment (DaveBirdsall) [inline docs/client_install/src/asciidoc/_chapters/dbviz.adoc]: I don't get it. The example uses 23400, and then the *NOTE* says we aren't using 23400.

File: docs/client_install/src/asciidoc/_chapters/jdbct4.adoc [hunk 2]
```diff
@@ -117,16 +80,42 @@ go to this link: http://www.java.com/en/download
...
+* <<jdbct4-path-linux, Setting the PATH to a Supported Java Version on Linux>>
 
-[[jdbct4_path_windows]]
+[[jdbct4-path-windows]]
 ==== Set Windows PATH Variable
 
-1.  Right-click the *Computer* icon on your desktop, and then select *Properties*. The *Control Panel > System and Security > System* window
+===== Windows 10
+
+1. Right-clock on the Windows icon on the menu bar. Select *System*.
```
Comment (DaveBirdsall) [inline docs/client_install/src/asciidoc/_chapters/jdbct4.adoc]: Typo, should be "Right-click".  (Though "Right-clock" sounds more interesting.)

File: docs/client_install/src/asciidoc/_chapters/dbviz.adoc (hunk 1 above)
Comment (gtapper) [inline docs/client_install/src/asciidoc/_chapters/dbviz.adoc]: You're not seeing the screenshot. :) It contains the 37800 value.

**Comment by DaveBirdsall** [comment]
It looks good, Gunnar. Thanks for doing this.

**Comment by gtapper** [comment]
Please hold off updating the website. I'm wrapping up the new SPJ Guide and will check it in Monday or Tuesday. No need to rebuild the website twice.

**Comment by gtapper** [comment]
Please process the update of the website. The new SPJ guide requires a bit more time before it can be checked in.

**Pull request merged**